__tests__/app.test.js

```diff
@@ -29,3 +30,37 @@ describe("GET /api/categories", () => {
     return request(app).get("/api/wrongEndpoint").expect(404);
   });
 });
+//
+describe("GET /api/reviews", () => {
+  it("200: should respond with review objects array, which should have 9 properties and sorted by date desc", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body).toBe("object");
+        expect(body).toHaveProperty("reviews");
+        const { reviews } = body;
+        expect(body.reviews).toBeInstanceOf(Array);
+        expect(reviews.length).toBe(13);
+        reviews.forEach((review) => {
+          expect(review).toHaveProperty("owner", expect.any(String));
+          expect(review).toHaveProperty("title", expect.any(String));
+          expect(review).toHaveProperty("review_id", expect.any(Number));
+          expect(review).toHaveProperty("category", expect.any(String));
+          expect(review).toHaveProperty("review_img_url", expect.any(String));
+          expect(review).toHaveProperty("created_at", expect.any(String));
+          expect(review).toHaveProperty("votes", expect.any(Number));
+          expect(review).toHaveProperty("designer", expect.any(String));
+          expect(review).toHaveProperty("comment_count", expect.any(String));
+        });
+        expect(reviews).toBeSortedBy("created_at", {
+          descending: true,
+        });
+      });
+  });
+  //
+  it("404: should respond with 404 not found", () => {
```

Comment:
dont forget the content of the lecture today, you won't need to add multiples of these tests and can now check the response rather than just the status code